minibroker/setup-bucket.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+set -e
+set -x
+
+s3cmd mb s3://minibroker-helm-charts
+
+aws s3api put-bucket-versioning --bucket minibroker-helm-charts --versioning-configuration Status=Enabled
```

Comment:
Hi @alexjh 

The `minibroker-helm-charts` constant seems to be the same as `s3-bucket` in the `config-production.yaml`, aka `MINIBROKER_BUCKET` in the pipeline config and scripts.

I believe that we should make sure that there cannot be a mismatch ... I.e. provide the config file to use as argument to this script (maybe default to config production), and use `awk` to pull the value of `s3-bucket` out of it. The syntax is simple enough for `awk` to work.

Reply:
Fixed in 784204c